client/src/components/global/Topbar.tsx

```diff
@@ -113,10 +114,15 @@ export default function Topbar(): JSX.Element {
 											Login
 										</Button>
 									) : (
-										<Button onClick={(): Promise<void> => validate(user)}
+										// <Button onClick={(): Promise<void> => validate(user)}
+										// 	color='secondary' startIcon={<PersonIcon />}
+										// 	variant='contained'>
+										// 	My Account
+										// </Button>
+										<Button onClick={(): Promise<void> => signout(user)}
 											color='secondary' startIcon={<PersonIcon />}
 											variant='contained'>
-											My Account
```

Comment:
So now we have a logout button, How will the user check their account? I think its a better idea to leave the account button and let it drop down to a few options. Check the figma designs I made a few for each possible gebruiker.

Reply:
it was for testing if it worked, forgot to remove it whoops